refl1d/probe/data_loaders/load4.py

```diff
@@ -87,26 +87,46 @@ def get_key(orso_name, refl1d_name, refl1d_resolution_name):
                 (i for i, c in enumerate(columns) if getattr(c, "physical_quantity", None) == orso_name),
                 None,
             )
+            resolution_index = None
             if column_index is not None:
                 # NOTE: this is based on column being second index (under debate in ORSO)
                 header_out[refl1d_name] = data[:, column_index]
                 cname = columns[column_index].name
-                resolution_index = next(
-                    (i for i, c in enumerate(columns) if getattr(c, "error_of", None) == cname),
-                    None,
+                resolution_index, resolution_column = next(
+                    ((i, c) for i, c in enumerate(columns) if getattr(c, "error_of", None) == cname),
+                    (None, None),
                 )
                 if resolution_index is not None:
                     header_out[refl1d_resolution_name] = data[:, resolution_index]
-            else:
-                v = getattr(settings, orso_name, None)
-                if hasattr(v, "magnitude"):
-                    header_out[refl1d_name] = v.magnitude
-                if hasattr(v, "error"):
-                    header_out[refl1d_resolution_name] = v.error.error_value
+                    if resolution_column.value_is == "FWHM":
+                        header_out[refl1d_resolution_name] = FWHM2sigma(header_out[refl1d_resolution_name])
+
+            # Fall back to instrument_settings if no column found
+            v = getattr(settings, orso_name, None)
+            if hasattr(v, "magnitude") and column_index is None:
+                # only set if not already set from column
+                header_out[refl1d_name] = v.magnitude
+            if hasattr(v, "error") and resolution_index is None:
+                header_out[refl1d_resolution_name] = v.error.error_value
+                if v.error.value_is == "FWHM":
+                    header_out[refl1d_resolution_name] = float(FWHM2sigma(header_out[refl1d_resolution_name]))
 
         get_key("incident_angle", "angle", "angular_resolution")
         get_key("wavelength", "wavelength", "wavelength_resolution")
 
+        # convert error columns from FWHM to sigma if they are present
+        # and need conversion
+        if len(columns) >= 3:
+            # third column is always uncertainty on reflectivity
+            dR_col = columns[2]
+            if dR_col.value_is == "FWHM":
+                data[:, 2] = FWHM2sigma(data[:, 2])
```

Comment:
This modifies the entry in place as returned by the orso/nexus reader. That's probably okay, but since you are copying the data anyway on line 130, you may as well move the copy to line 62 so that you are operating on your own copy of the data. You might also force the data type to be float64 when you create the copy.

I'm a little concerned that ΔR is potentially being scaled by FWHM2sigma. Is there any risk that the ΔR column is mislabeled as FWHM, e.g., because it is defaulting to FWHM when the value isn't specified? This would be happening in the orso/nexus readers.